templates/taxi/car_form.html

```diff
@@ -6,7 +6,6 @@ <h1>{{ object|yesno:"Update,Create" }} car</h1>
   <form action="" method="post" novalidate>
     {% csrf_token %}
     {{ form|crispy }}
-
     <input type="submit" value="Submit" class="btn btn-primary">
   </form>
 {% endblock %}
```

Comment:
https://stackoverflow.com/questions/729692/why-should-text-files-end-with-a-newline